src/screens/UserPortal/UserGlobalScreen/UserGlobalScreen.spec.tsx

```
@@ -16,12 +24,47 @@ vi.mock('components/UserPortal/UserSidebar/UserSidebar', () => ({
-vi.mock('components/ProfileDropdown/ProfileDropdown', () => ({
-  default: vi.fn(() => (
-    <div data-testid="profile-dropdown">ProfileDropdown</div>
+vi.mock('shared-components/DropDownButton', () => ({
+  default: vi.fn((props) => (
+    <div
+      data-testid="user-profile-dropdown"
+      data-variant={props.variant}
+      data-menu-class={props.menuClassName}
+      data-show-caret={props.showCaret}
+      aria-label={props.ariaLabel}
+    >
```

Biome (2.3.13)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)